### PR TITLE
Isomorphisms are equal if the underlying morphisms are equal

### PR DESCRIPTION
**Pull request by jcranch**
Previously this was just a postulate (with an extra assumption, too)

### CODE REVIEW

File: src/Basic/Isomorphism.lidr [hunk 1]
```diff
@@ -61,6 +62,35 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 >   inverse
 >   (MkInverseMorphisms leftInverse rightInverse)
 >
+> isomorphismEq :
+>      {cat : Category}
+>   -> {a, b : obj cat}
+>   -> (hom : mor cat a b)
```
Comment (marcosh) [inline src/Basic/Isomorphism.lidr]: what do you say about making this `hom` argument implicit?

**Comment by marcosh** [comment]
very nice @jcranch! removing a `postulate` is always nice. It was there for historical reasons, I guess, before we splitted `InverseMorphisms`, `Isomorphism` and `Isomorphic`

File: src/Basic/Isomorphism.lidr (hunk 1 above)
Comment (jcranch) [reply src/Basic/Isomorphism.lidr]: Can do! I mean to email you about implicits at some point.

**Comment by marcosh** [comment]
I made that `hom` argument implicit and merged this. Thanks again @jcranch

**Pull request closed without merging**